docs/toc.html

```diff
@@ -50,13 +50,23 @@
                         <li><a href="#sourceconnectconfigs">Source Connector Configs</a>
                         <li><a href="#sinkconnectconfigs">Sink Connector Configs</a>
                     </ul>
+                
                 <li><a href="#streamsconfigs">3.6 Kafka Streams Configs</a>
                 <li><a href="#adminclientconfigs">3.7 AdminClient Configs</a>
                 <li><a href="#mirrormakerconfigs">3.8 MirrorMaker Configs</a>
                 <li><a href="#systemproperties">3.9 System Properties</a>
-                <li><a href="#tieredstorageconfigs">3.10 Tiered Storage Configs</a></li>
+                <li><a href="#tieredstorageconfigs">3.10 Tiered Storage Configs</a>
+                <li><a href="#config_providers">3.11 Configuration Providers</a>
+                    <ul>
+                        <li><a href="#using_providers">Using</a>
```

Comment:
Should this be `Using Configuration Providers` to match the section title?

Reply:
I think it's fine as it is. I tried a full title with this and referencing and they looked too long in the toc and the concept gets a bit lost. 

![image](https://github.com/user-attachments/assets/dc510ce4-669f-4fdf-a0bf-9cd62fd5f714)

with full names:

![image](https://github.com/user-attachments/assets/4b4153dc-02c4-4714-9c41-4c202cd5fcd5)

Reply:
I'm not a native English speaker and just "Using" looks weird to me. If we don't want the full title because it's too long, would "Usage" work? If you think "Using" is fine, then I'll trust you.

Reply:
Ok, thanks for the explanation